### ISSUE TITLE
Dependency 'xdotool' not available when installing SendCode

### ISSUE THREAD

**Issue by Richyrecarey** [randy3k/SendCode#86]
Hi, I'm using SendCode+SublimeREPL R on macOS and everything works well, but when trying to install SendCode on Ubuntu 17.10 I came across this dependencis error (SublimeREPL does work): 

**ERROR:**

startup, version: 3143 linux x64 channel: stable
executable: /opt/sublime_text/sublime_text
working dir: /
packages path: /home/richyrecarey/.config/sublime-text-3/Packages
state path: /home/richyrecarey/.config/sublime-text-3/Local
zip path: /opt/sublime_text/Packages
zip path: /home/richyrecarey/.config/sublime-text-3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.085186
startup time: 0.149914
first paint time: 0.155177
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.install_package_control
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
reloading plugin SublimeREPL.__init__
reloading plugin SublimeREPL.completions
reloading plugin SublimeREPL.lang_integration
reloading plugin SublimeREPL.run_existing_command
reloading plugin SublimeREPL.sublimerepl
reloading plugin SublimeREPL.sublimerepl_build_system_hack
reloading plugin SublimeREPL.text_transfer
plugins loaded
reloading settings Packages/User/Package Control.sublime-settings
Package Control: No updated packages
ignored packages updated to: ["SendCode", "Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
Package Control: The dependency 'xdotool' is not currently installed; installing...
Package Control: The dependency 'xdotool' is not available
Package Control: The dependency 'xdotool' could not be installed or updated
ignored packages updated to: ["Vintage"]
reloading settings Packages/User/Package Control.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings
reloading settings Packages/User/Preferences.sublime-settings

This is a fresh Sublime Text 3 install so I dont know what the package Vintage means, or how can I install the missed dependency "xdotool". I've done:

```
sudo apt-get update
sudo apt-get install xdotool 
```

without success.

Thank you in advance!

**Comment by randy3k** [comment]
Please try again. Package control should install the dependencies automatically.

**Comment by Richyrecarey** [comment]
I did everything again without success, trying different approaches to install SB3. Is there any other way to install package control without doing it from within sublime text? 

Parallelly, I also downloaded the SB3 folder from the webpage (not through apt-get), and end up with the same 3 errors. Same happens, for example, trying to install R-box package, but with a different dependency problem:

Package Control: The dependency 'pygments' is not currently installed; installing...
Package Control: The dependency 'pygments' is not available
Package Control: The dependency 'pygments' could not be installed or updated

Thank you.
